app.cpp

```diff
@@ -161,7 +186,22 @@ class App {
             m_ui_state &= ~Ui::State::Warning;
         }
 
-        m_ui.set_big_number(static_cast<uint8_t>(round(current_temperature)));
+        if (brew_sensor.is_connected()) {
+            m_ui.set_big_number(static_cast<uint8_t>(round(current_temperature)));
+        } else {
+            m_ui.set_big_number(0);
+        }
+        if (ambient_sensor.is_connected()) {
+            m_ui.set_tiny_number1(static_cast<uint8_t>(round(ambient_temperature)));
+        } else {
+            m_ui.set_tiny_number1(0);
+        }
+        if (sparging_sensor.is_connected()) {
+            m_ui.set_tiny_number2(static_cast<uint8_t>(round(sparging_temperature)));
+        } else {
+            m_ui.set_tiny_number2(0);
+        }
```

Comment:
I am not yet convinced of the UI/UX of this change. It starts getting noisy/cluttered again for very little gain. All temperatures have a certain importance at certain times but not all at once. Moreover, we will have this information on the controller side anyway.

IMHO, a button press to cycle through the temperatures would make more sense. Should we maybe ask Mrs. UX?

Reply:
It was never my intention to keep this in the final version. But such snippets (as well as the refresh indicator) help amateur me during development, and I might want to occasionaly reuse those in the future. 

What would be the best workflow for such things? Local branch and rebase all the time?